src/app/Components/Artist/ArtistHeaderNavRight.tsx

```diff
@@ -21,10 +20,8 @@ export const ArtistHeaderNavRight: React.FC<ArtistHeaderNavRightProps> = ({
   const data = useFragment(fragment, artist)
   const [isFollowed, setIsFollowed] = useState(!!data?.isFollowed)
 
-  const { handleFollowToggle } = useFollowArtist({
-    artist: data,
-    ownerType: Schema.OwnerEntityTypes.Artist,
-  })
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const { handleFollowToggle } = useFollowArtist(data!)
```

Comment:
Instead, we could allow `data` to be `null` in `useFollowArtist`.
```suggestion
  const { handleFollowToggle } = useFollowArtist(data)
```

Reply:
True, it actually has already been addressed and i changed it back. Thanks!